button-examples/Android/android-button-example.md

```diff
@@ -42,6 +48,71 @@ The following example shows a text button with a text label.
     />
 ```
 
+### Key properties
+
+<details>
+<summary><b>Styles</b></summary>
+<p>
+
+Description | Style
+---|---
+Default style | `Widget.MaterialComponents.Button.TextButton`
+Icon style | `Widget.MaterialComponents.Button.TextButton.Icon`
+
+</p>
+</details>
+
+<details>
```

Comment:
GitHub-flavored MarkDown doesn't seem to like nesting collapsible sections -- it's not rendering properly.

Reply:
NM, didn't see the render, though I'm going to see if the nesting can be indented.